x-pack/plugins/data_visualizer/public/application/common/components/document_count_content/document_count_content.tsx

```diff
@@ -30,14 +77,115 @@ export const DocumentCountContent: FC<Props> = ({ documentCountStats, totalCount
     chartPoints = Object.entries(buckets).map(([time, value]) => ({ time: +time, value }));
   }
 
+  const approximate = documentCountStats.randomlySampled === true;
+
+  const ProbabilityUsed =
+    randomSamplerPreference !== RANDOM_SAMPLER_OPTION.OFF && isDefined(samplingProbability) ? (
+      <FormattedMessage
+        id="xpack.dataVisualizer.randomSamplerSettingsPopUp.probabilityLabel"
+        defaultMessage="Probability used: {samplingProbability}%"
+        values={{ samplingProbability: samplingProbability * 100 }}
+      />
+    ) : null;
   return (
     <>
-      <TotalCountHeader totalCount={totalCount} />
+      <EuiFlexGroup alignItems="center" gutterSize="xs">
+        <TotalCountHeader totalCount={totalCount} approximate={approximate} loading={loading} />
+        <EuiFlexItem grow={false}>
+          <EuiPopover
+            id="dscSamplingOptions"
+            button={
+              <EuiToolTip
+                content={i18n.translate('xpack.dataVisualizer.samplingOptionsButton', {
+                  defaultMessage: 'Sampling options',
+                })}
+              >
+                <EuiButtonIcon
+                  size="xs"
+                  iconType="gear"
+                  onClick={onShowSamplingOptions}
+                  data-test-subj="discoverSamplingOptionsToggle"
+                  aria-label={i18n.translate('xpack.dataVisualizer.samplingOptionsButton', {
+                    defaultMessage: 'Sampling options',
+                  })}
+                />
+              </EuiToolTip>
+            }
+            isOpen={showSamplingOptionsPopover}
+            closePopover={closeSamplingOptions}
+            panelPaddingSize="none"
+            anchorPosition="downLeft"
+          >
+            <EuiPanel style={{ maxWidth: 400 }}>
+              <EuiFlexItem grow={true}>
+                <EuiCallOut
+                  size="s"
+                  color={'primary'}
+                  title={i18n.translate(
+                    'xpack.dataVisualizer.randomSamplerSettingsPopUp.infoCalloutMessage',
+                    {
+                      defaultMessage:
+                        'Random sampler is being used for the total document count and the chart. Pick a higher percentage for better accuracy, or "Off" for no sampling.',
+                    }
+                  )}
+                />
+              </EuiFlexItem>
+              <EuiSpacer size="m" />
+
+              <EuiSelect
```

Comment:
Could be worth putting this control in an `EuiFormRow` and associating a label `Random sampling` with it, to make it clear exactly what is 'on' or 'off (and also for accessibility):

<img width="347" alt="image" src="https://user-images.githubusercontent.com/7405507/180265121-bbcc26f8-caba-48ef-839e-865e8321c78f.png">